envoy.dependency.check/envoy/dependency/check/abstract/issues.py

```diff
@@ -28,7 +28,8 @@
 Upstream releases: https://github.com/{full_name}/releases
 New Issue Link: https://github.com/{repo_location}/issues/{number}
 """
-ISSUES_SEARCH_TPL = "in:title {self.title_prefix} is:open is:issue"
+ISSUES_SEARCH_TPL = "in:title {self.title_prefix} is:open " + \
```

Comment:
please use brackets split text lines rather than `\`

Reply:
also - please please please - conjunctions (ie `+`) at the beginning of the line

you wont need it tho, if you wrap in brackets